op-node/metrics/metrics.go

```diff
@@ -111,6 +120,18 @@ type Metrics struct {
 	SequencerSealingDurationSeconds prometheus.Histogram
 	SequencerSealingTotal           prometheus.Counter
 
+	SequencerBuilderRequestDurationSeconds prometheus.Histogram
+	SequencerBuilderRequestTotal           prometheus.Counter
+	SequencerBuilderRequestErrors          prometheus.Counter
+	SequencerBuilderRequestTimeouts        prometheus.Counter
+
+	SequencerProfit      *prometheus.GaugeVec
+	SequencerProfitTotal *prometheus.CounterVec
```

Comment:
can `CounterVec` handle float64 values? Shouldn't this be an increasing `prometheus.GaugeVec`